pkg/mcs/resource_manager/server/config.go

```diff
@@ -32,13 +32,15 @@ import (
 
 const (
 	defaultName              = "Resource Manager"
-	defaultBackendEndpoints  = "127.0.0.1:2379"
+	defaultBackendEndpoints  = "http://127.0.0.1:3379"
```

Comment:
maybe conflict with tso's `defaultListenAddr`

Reply:
BTW, I think 2379 is better.